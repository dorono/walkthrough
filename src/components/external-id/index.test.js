import React from 'react';
import {shallow} from 'enzyme';
import ExternalId from './index';

const defaultProps = {
    data: [],
};

describe('ExternalId', () => {
    it('should render without errors', () => {
        shallow(<ExternalId {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<ExternalId {...defaultProps} />)).toMatchSnapshot();
    });
});
