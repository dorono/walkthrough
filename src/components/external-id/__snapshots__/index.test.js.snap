// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalId should match snapshot 1`] = `
<ExternalId
  data={
    Array [
      Object {
        "label": "raw",
        "value": "",
      },
    ]
  }
  showDefaultEncoding={false}
/>
`;
